cnf-certification-test/results/html/results.html

```diff
@@ -5,7 +5,7 @@
   <title>CNF Certification Test Results</title>
   <meta charset="utf-8">
   <meta name="viewport" content="width=device-width, initial-scale=1">
-  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.1/dist/css/bootstrap.min.css" integrity="sha256-2TnSHycBDAm2wpZmgdi0z81kykGPJAkiUY+Wf97RbvY=" crossorigin="anonymous">
```

Comment:
this file should be deleted. It is created by the tnf exe after running the tests